src-db/database/model/tables/ETDEP_DEPENDENCY.xml

```diff
@@ -69,6 +69,10 @@
         <default><![CDATA[PENDING]]></default>
         <onCreateDefault><![CDATA[PENDING]]></onCreateDefault>
       </column>
+      <column name="ISEXTERNALDEPENDENCY" primaryKey="false" required="true" type="CHAR" size="1" autoIncrement="false">
+        <default><![CDATA[N]]></default>
+        <onCreateDefault/>
```

Comment:
For tables already populated with data, the creation of a new mandatory column should include a value set for it in the onCreateDefault property